src/Components/LeftSideWeatherApp/CurrentDayWeatherForecast/CurrentDayWeatherForecast.js

```diff
@@ -1,23 +1,24 @@
 import './CurrentDayWeatherForecast.css';
                                   
 function CurrentDayWeatherForecast(props) {
-    const weekday = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+    const weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
     const d = new Date();
     let day = weekday[d.getDay()];
 
     return (
         <div className='current-day-weather-forecast'>
-            <img className='weather-image-icon' src={props.iconUrl} alt="weather-icon"/>
             {
                 props.weatherData.map(data => {
                     return (
                         <div className='weather-forecast'>
```

Comment:
you should always assign a `key` to components or elements rendered in a loop like a `.map`. Read up on react keys. There will be errors in your dev console telling you of this already.